### PR TITLE
Modify performance tests

### PR DESCRIPTION
**Pull request by mykhailo-kuchma**
* Remove obsolete nullcache tests.
* Remove old memory prefetch tests.
* Add 15m stability tests.

Relates-To: OLPEDGE-2424

Signed-off-by: Mykhailo Kuchma <ext-mykhailo.kuchma@here.com>

### CODE REVIEW

File: tests/performance/MemoryTestBase.h [hunk 1]
```diff
@@ -22,13 +22,13 @@
 #include <gtest/gtest.h>
 
 #include <olp/core/cache/CacheSettings.h>
+#include <olp/core/cache/KeyValueCache.h>
 #include <olp/core/client/OlpClientSettings.h>
 #include <olp/core/client/OlpClientSettingsFactory.h>
 #include <olp/core/utils/Dir.h>
 #include <testutils/CustomParameters.hpp>
 
 #include "NetworkWrapper.h"
```
Comment (andescu) [inline tests/performance/MemoryTestBase.h:31]: If you already touch this, then please do it properly. All headers should be grouped together as none are system headers.

File: tests/performance/MemoryTest.cpp [hunk 2]
```diff
@@ -197,76 +196,19 @@ TEST_P(MemoryTest, ReadNPartitionsFromVersionedLayer) {
   });
 }
 
-TEST_P(MemoryTest, PrefetchPartitionsFromVersionedLayer) {
```
Comment (andescu) [inline tests/performance/MemoryTest.cpp:200]: Is there a reason why you removed this.

Comment (mykhailo-kuchma) [reply tests/performance/MemoryTest.cpp:200]: An old test that we don't use, we have another one is a separate file.

Comment (andescu) [reply tests/performance/MemoryTest.cpp:200]: Ok.

**Review by andescu — APPROVED** [review]

**Review by meravingen123 — APPROVED** [review]

**Pull request merged**